modules/dnn/cmake/download_google_inception5.cmake

```diff
@@ -0,0 +1,30 @@
+function(download_google_inception5 dst_dir status_var)
```

Comment:
Hi, @abratchik ! `OPENCV_TEST_DATA_PATH` and `OPENCV_DNN_TEST_DATA_PATH` both works only as environment variables.